smoke_screens/product_show_screen.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'basic_show_screen'
+
+module SmokeScreens
+  class ProductShowScreen < BasicShowScreen
+    def parent_page_url
+      '/widgets/products'
```

Comment:
This is widget page and not show page

the show page is  /products/:product_id 

but it's good to test the widget page too

Reply:
@jwachira The logic of any show page is based on taking of the target 'show' link from any page which contains links on products. We use this because the list of available products can be changed and we can't use any preferred product for a test without using the pre-created test product. It's also true for any other 'show' tests.

Reply:
@jeyboy understood